### PR TITLE
Update's our syndicate multitools to be in line with /tg/'s

### PR DESCRIPTION
**Pull request by Spacemanspark**
Have someone look through this first.
This is /tg/'s presumably updated syndicate multitool code.

### CODE REVIEW

File: code/game/objects/items/devices/multitool.dm [hunk 1]
```diff
@@ -33,29 +33,34 @@
 
 /obj/item/device/multitool/ai_detect/New()
 	..()
-	processing_objects += src
-
+	SSobj.processing += src
```
Comment (tigercat2000) [inline code/game/objects/items/devices/multitool.dm]: Not going to work.

File: code/game/objects/items/devices/multitool.dm [hunk 2]
```diff
@@ -33,29 +33,34 @@
...
 /obj/item/device/multitool/ai_detect/New()
 	..()
-	processing_objects += src
-
+	SSobj.processing += src
 
 /obj/item/device/multitool/ai_detect/Destroy()
-	processing_objects -= src
-	return ..()
+	SSobj.processing -= src
```
Comment (tigercat2000) [inline code/game/objects/items/devices/multitool.dm]: Not going to work.

File: code/game/objects/items/devices/multitool.dm [hunk 3]
```diff
@@ -33,29 +33,34 @@
...
+	..()
```
Comment (tigercat2000) [inline code/game/objects/items/devices/multitool.dm]: Needs to be return ..()

File: code/game/objects/items/devices/multitool.dm [hunk 4]
```diff
@@ -33,29 +33,34 @@
...
+		if(AI.cameraFollow == src)
 			found_eye = 1
 			break
 
+	if(!found_eye && cameranet.chunkGenerated(our_turf.x, our_turf.y, our_turf.z))
+		var/datum/camerachunk/chunk = cameranet.getCameraChunk(our_turf.x, our_turf.y, our_turf.z)
+		if(chunk)
+			if(chunk.seenby.len)
+				for(var/mob/camera/aiEye/A in chunk.seenby)
+					var/turf/eye_turf = get_turf(A)
```
Comment (tigercat2000) [inline code/game/objects/items/devices/multitool.dm]: I doubt any of the will work, too.

**Comment by Spacemanspark** [comment]
Oh come on it compiled when I did it locally.
Welp, I should stop trying to do this kind of thing at 1 AM.

**Pull request closed without merging**